### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by Vvidsky**
This feature is for adding an autocomplete into the search box

### CODE REVIEW

**Comment by ChanAndKlee** [comment]
Can you complete this function by today?

File: lab-exercise-1/KulawutFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class KulawutFeature {
+	public void feature() {
+		// this is a new feature
```
Comment (ChanAndKlee) [inline lab-exercise-1/KulawutFeature.java:3]: This function is incomplete

**Review by ChanAndKlee — COMMENTED** [review]
Please complete by today

**Review by Jack5hj — COMMENTED** [review]
this need be improved ty

File: lab-exercise-1/KulawutFeature.java (hunk 1 above)
Comment (Pontuspts) [inline lab-exercise-1/KulawutFeature.java:3]: This content is not well-written. Please improve it.

**Comment by NoodlesCupCake** [comment]
This content is not well-written. Please improve it.

File: lab-exercise-1/KulawutFeature.java (hunk 1 above)
Comment (NoodlesCupCake) [reply lab-exercise-1/KulawutFeature.java:3]: This content isn't well-written, please improve it.

[the remark this answers is not in the document]
Comment (NoodlesCupCake) [reply lab-exercise-1/KulawutFeature.java:3]: This content isn't well-written, please improve it.

Comment (NoodlesCupCake) [reply lab-exercise-1/KulawutFeature.java:3]: This content isn't well-written, please improve it.

Comment (KRoos555) [reply lab-exercise-1/KulawutFeature.java:3]: This contents is not well-written. Please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
nice

**Review by Pontuspts — COMMENTED** [review]
nice

File: lab-exercise-1/KulawutFeature.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
-public class KulawutFeature {
+public class <Yourname>Feature {
 	public void feature() {
-		// this is a new feature
+ 		// this is a new feature
```
Comment (Vvidsky) [inline lab-exercise-1/KulawutFeature.java:3]: The new fix is available now.

File: lab-exercise-1/KulawutFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+ 		int x = 0;
+ 		x = x+1;
```
Comment (Vvidsky) [inline lab-exercise-1/KulawutFeature.java:5]: The new fix is available now

**Review by NoodlesCupCake — APPROVED** [review]
very good, testing

File: lab-exercise-1/KulawutFeature.java (hunk 3 above)
Comment (NoodlesCupCake) [reply lab-exercise-1/KulawutFeature.java:5]: good makx2

File: lab-exercise-1/KulawutFeature.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-public class <Yourname>Feature {
+public class KulawutFeature {
```
Comment (Vvidsky) [inline lab-exercise-1/KulawutFeature.java:1]: The feature is completed

**Pull request merged**